.env.test

```diff
@@ -1,4 +1,4 @@
 # define your env variables for the test env here
 KERNEL_CLASS='App\Kernel'
 APP_SECRET='s$cretf0rt3st'
-SYMFONY_DEPRECATIONS_HELPER=999999
+SYMFONY_DEPRECATIONS_HELPER='max[self]=0'
```

Comment:
It looks like this configuration is not handled correctly:
>InvalidArgumentException: Unknown configuration option "'max"

Maybe we could remove it from here and use `<env name="SYMFONY_DEPRECATIONS_HELPER" value="max[self]=0" />` in phpunit.xml.dist?

Reply:
The problem was in circle ci config, but I implemented your suggestion anyway.